hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationService.java

```diff
@@ -123,7 +124,7 @@
 
     private final long defaultCallTimeoutMillis;
     private final long backupOperationTimeoutMillis;
-    private final OperationHandler operationHandler;
+    private final BasicOperationHandler operationHandler;
     private final OperationBackupHandler operationBackupHandler;
     private final BasicBackPressureService backPressureService;
     private final ResponsePacketHandler responsePacketHandler;
```

Comment:
Don't look too much at these details for the time being :) Just see if the API fits your needs and reduces the current thight coupling to the scheduler threads.